nnpdf_data/nnpdf_data/commondata/ATLAS_Z0_7TEV_46FB/metadata.yaml

```diff
@@ -19,7 +19,7 @@ implemented_observables:
     label: ATLAS $Z$ 7 TeV 2011 Central selection
     units: ''
   process_type: EWK_RAP
-  tables: []
+  tables: [11-13]
```

Comment:
this should be [11, 12, 13]. Then you can use it to loop over in eg `filter_utils/get_data_values` instead of having to write three times 

```
with open(...
```

Reply:
Yes, `tables` should be a Python list specifying the table ID. This will be crucial when we want to remove the rawdata from the remote repository (cc https://github.com/NNPDF/nnpdf/issues/2199).